### PR TITLE
修复非主机业务资源下点击列表排序无法排序问题 issue #5139

### PR DESCRIPTION
**Pull request by chalice-1831**
bugfix 修复非主机业务资源下点击列表排序无法排序问题

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-cmdb?pullRequest=5117) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-cmdb?pullRequest=5117) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-cmdb?pullRequest=5117) it.</sub>

**Comment by chalice-1831** [comment]
#5139

**Comment by chalice-1831** [comment]
#5139 v3.5.x 更换接口以及功能修复

File: src/scene_server/datacollection/pid/datacollection.exe.pid [hunk 1]
```diff
@@ -0,0 +1 @@
+16344
```
Comment (breezelxp) [inline src/scene_server/datacollection/pid/datacollection.exe.pid:1]: 这几个pid文件是什么文件? 这不能提上来吧？

**Review by breezelxp — CHANGES_REQUESTED** [review]

**Review by breezelxp — APPROVED** [review]

**Pull request merged**